src/Propulsion/Sinks.fs

```diff
@@ -17,36 +17,9 @@ type Codec<'E> = FsCodec.IEventCodec<'E, EventBody, unit>
 module Events =
 
     /// The Index of the next event ordinarily expected on the next handler invocation (assuming this invocation handles all successfully)
-    let nextIndex: Event[] -> int64 = Streams.StreamSpan.ver
+    let nextIndex: Event[] -> int64 = Streams.StreamSpan.nextIndex
     /// The Index of the first event as supplied to this handler
-    let index: Event[] -> int64 = Streams.StreamSpan.idx
-
-/// Represents progress attained during the processing of the supplied Events for a given <c>StreamName</c>.
-/// This will be reflected in adjustments to the Write Position for the stream in question.
-/// Incoming <c>StreamEvent</c>s with <c>Index</c>es prior to the Write Position implied by the result are proactively
-/// dropped from incoming buffers, yielding increased throughput due to reduction of redundant processing.
-type StreamResult =
```

Comment:
Moving this is a breaking change

Reply:
it's only a RC 😢

Reply:
Further, the original transition to this name/structure happened in V3 and I consider it a bug - my thinking was that the name was something to hang documentation off.

On reflection, confronting people with an `int64` makes a lot more sense - you have to think, and you don't have to explain the `OverrideNextIndex` name